packages/styling/src/styles/DefaultEffects.ts

```diff
@@ -0,0 +1,10 @@
+import { IEffects } from '../interfaces/index';
+
+export const DefaultEffects: IEffects = {
+  elevation4: { boxShadow: '0 0 5px 0 rgba(0,0,0,.4)' }, // '0 1.6px 3.6px 0 rgba(0, 0, 0, 0.132), 0 0.3px 0.9px 0 rgba(0, 0, 0, 0.108)'
+  elevation8: { boxShadow: '0 0 5px 0 rgba(0,0,0,.4)' }, // '0 3.2px 7.2px 0 rgba(0, 0, 0, 0.132), 0 0.6px 1.8px 0 rgba(0, 0, 0, 0.108)'
+  elevation16: { boxShadow: '0 0 5px 0 rgba(0,0,0,.4)' }, // '0 6.4px 14.4px 0 rgba(0, 0, 0, 0.132), 0 1.2px 3.6px 0 rgba(0, 0, 0, 0.108)'
+  elevation64: { boxShadow: '0 0 5px 0 rgba(0,0,0,.4)' }, // '0 25.6px 57.6px 0 rgba(0, 0, 0, 0.22), 0 4.8px 14.4px 0 rgba(0, 0, 0, 0.18)'
```

Comment:
Per our chat, I think a brief descriptive comment that commented values are Fluent values that will eventually be default may reduce confusion for anybody who comes across this later (as I was wondering why the differed.)